aptos-move/aptos-transactional-test-harness/src/aptos_test_harness.rs

```diff
@@ -328,10 +332,17 @@ static PRECOMPILED_APTOS_FRAMEWORK_V2: Lazy<PrecompiledFilesModules> = Lazy::new
         .map(|(string, num_addr)| format!("{}={}", string, num_addr))
         .collect();
 
+    let source_packages = vec![PackagePaths {
+        name: None,
+        paths: string_vec_to_symbol_vec(
+            &aptos_cached_packages::head_release_bundle()
+                .files()
+                .unwrap(),
+        ),
+        named_address_map: string_map_to_symbol_map(aptos_framework::named_addresses()),
+    }];
     let options = move_compiler_v2::Options {
-        sources: aptos_cached_packages::head_release_bundle()
-            .files()
-            .unwrap(),
+        packages: source_packages,
```

Comment:
I really would prefer we would not have v1 stuff like PackagePaths leak to the v2 code base. The current driver was intentionally build to not use this type. Why do we need it now?